commit 1fa52261f7255bf5e44b337d1605c48ea356eb7b
Author: Slávek Banko <slavek.banko@axis.cz>
Date:   Thu Oct 24 02:18:46 2024 +0200

    kxsldbg: Remove obsolete initialization of SAX1 parser.
    The call does not do any operation and causes FTBFS with libxml2 >= 2.13.0.
    
    Signed-off-by: Slávek Banko <slavek.banko@axis.cz>

diff --git a/kxsldbg/kxsldbgpart/libxsldbg/xsldbg.cpp b/kxsldbg/kxsldbgpart/libxsldbg/xsldbg.cpp
index d05281f5..8b1e7063 100644
--- a/kxsldbg/kxsldbgpart/libxsldbg/xsldbg.cpp
+++ b/kxsldbg/kxsldbgpart/libxsldbg/xsldbg.cpp
@@ -1220,12 +1220,6 @@ xsldbgInit()
         xsltSetGenericErrorFunc(0, xsldbgGenericErrorFunc);
 #endif
 
-	/*
-	 * disable CDATA from being built in the document tree
-	 */
-	xmlDefaultSAXHandlerInit();
-	xmlDefaultSAXHandler.cdataBlock = NULL;
-
         if (getThreadStatus() != XSLDBG_MSG_THREAD_NOTUSED) {
             initialized = 1;
             return 1;           /* this is all we need to do when running as a thread */
